packages/block-library/src/query/deprecated.js

```diff
@@ -296,6 +296,80 @@ const v3 = {
 	},
 };
 
-const deprecated = [ v3, v2, v1 ];
+const v4 = {
```

Comment:
Can we add a fixture to match this added deprecation? (e.g. `core__query__deprecated-4.html` etc)
It might be a little tricky if the deprecation depends on an earlier version of `useInnerBlocksProps.save()` to make sure that we test the deprecation is being run correctly 🤔

Reply:
Thanks, I always forget about the fixtures 😅